src/utilcode/util.cpp

```diff
@@ -1219,16 +1219,32 @@ int GetCurrentProcessCpuCount()
             count = 64;
     }
 
+    cCPUs = count;
+
+    return count;
+}
+
+int GetCurrentProcessCpuLimit()
+{
 #ifdef FEATURE_PAL
+    static int bCPUs = 0;
```

Comment:
Just curious - what does the "b" prefix in the bCPUs mean?

Reply:
`budget`, it's a leftover from the previous naming. I was trying to match `cCPUs` from `GetCurrentProcessCpuCount`.

Reply:
This naming style is Hungarian notation: https://en.wikipedia.org/wiki/Hungarian_notation

`b` in Hungarian stands for `byte` or `bool` that is not quite right here.

Reply:
Oh, good to know.